Formula/h/hackernews-tui.rb

```diff
@@ -0,0 +1,23 @@
+class HackernewsTui < Formula
+  desc "Terminal UI to browse Hacker News"
+  homepage "https://github.com/aome510/hackernews-TUI"
+  url "https://github.com/aome510/hackernews-TUI/archive/refs/tags/v0.13.4.tar.gz"
+  sha256 "77abc579f58ed33905f00eb4d6047b1d7f30a0b4ed98b44b36ff95648c6ebae0"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(root: ".", path: "hackernews_tui")
+    bin.install "bin/hackernews_tui" => "hackernews-tui"
+  end
+
+  test do
+    system "#{bin}/hackernews-tui", "--version"
```

Comment:
```suggestion
    system "#{bin}/hackernews_tui", "--version"

    pid = fork do
      exec "#{bin}/hackernews_tui"
      sleep 3
    ensure
      Process.kill("TERM", pid)
    end
```

Reply:
This will at least let us run the binary and ensure it doesn't immediately crash.

Reply:
Thanks, I will add this. If we are doing this, we could also just open a story, `hackernews-tui -i 8863`